<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCLVisualizerClass</class>
 <widget class="QMainWindow" name="PCLVisualizerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>937</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCLVisualizer</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QVTKWidget" name="qvtkWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件(F)</string>
    </property>
    <addaction name="OpenFile"/>
    <addaction name="Exit"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>设置(S)</string>
    </property>
    <addaction name="SetColor"/>
    <addaction name="OpenProjectManage"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_S"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_ProjectManageWindows">
   <property name="minimumSize">
    <size>
     <width>89</width>
     <height>111</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>项目管理</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTreeWidget" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string>工程目录</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="OpenFile">
   <property name="icon">
    <iconset>
     <normaloff>icon/fileopen.png</normaloff>icon/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>打开(O)</string>
   </property>
  </action>
  <action name="Exit">
   <property name="text">
    <string>退出(E)</string>
   </property>
  </action>
  <action name="SetColor">
   <property name="text">
    <string>颜色(C)</string>
   </property>
  </action>
  <action name="OpenProjectManage">
   <property name="text">
    <string>项目管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="pclvisualizer.qrc"/>
 </resources>
 <connections/>
</ui>
